frontend_tests/puppeteer_tests/settings.ts

```diff
@@ -289,16 +280,14 @@ async function delete_alert_word(page: Page, word: string): Promise<void> {
 
 async function test_alert_word_deletion(page: Page, word: string): Promise<void> {
     await delete_alert_word(page, word);
-    const status_text = await get_alert_words_status_text(page);
-    assert.strictEqual(status_text, "Alert word removed successfully!");
-    await close_alert_words_status(page);
```

Comment:
We still show the message when deleting the alert word, right? so why remove this?

Reply:
Fixed it. Thanks.